### ISSUE TITLE
Label truncated

### ISSUE THREAD

**Issue by coliss86** [MarginallyClever/Makelangelo-software#508]
**Steps to make the bug happen**
1. Load a big image
2. Apply A CMYK transformation
3. The label on the lower right of the slider is truncated
![image](https://user-images.githubusercontent.com/23615562/152232582-dc53a744-b926-49d6-ade2-1cb130e3c0b1.png)

**What was supposed to happen**
It should not be truncated

**Platform (please complete the following information):**
 - Version: master
 - OS: macos

**Log file**
none

**Comment by i-make-robots** [comment]
I set fixed values for the number and had no idea how many would be there.  If I let the system dynamically choose the best size a new problem appears: as I drag the slider the field size changes, which affects the slider length, which moves the slider head on the slider.  so it can happen that you're trying to drag to a very specific number, the slider jumps, and you CAN'T hit the number you want.

**Comment by i-make-robots** [comment]
I think the best solution is to allow the fields to grow but not to shrink.  so they might look funny if there's a small number in a large space... but it will work with minimum user friction.

**Comment by i-make-robots** [comment]
Minimum size is ignored by something in the system.  Maybe the BorderLayout?  In any case it doesn't work.  Preferred size and maximum size don't work either, number truncated.

**Comment by coliss86** [comment]
The latest solution is to pad on the right the number with space. So every time, the label is at the correct size

**Comment by i-make-robots** [comment]
I doubt spaces are the same with as digits in all fonts.

**Comment by coliss86** [comment]
Defining a fixed size font will ensure it. Something like : 
```
.setFont(new Font(Font.MONOSPACED, Font.PLAIN, 12));
```

**Comment by coliss86** [comment]
The problem is still present. The number is `351228`
![image](https://user-images.githubusercontent.com/23615562/153237589-3fcebcbc-19b6-4d9d-94d6-e755b3cea773.png)

By setting the font and increasing a little the space, it solves the problem : 
![image](https://user-images.githubusercontent.com/23615562/153238101-f807acdb-edf2-43fb-8ed3-5d722b1db820.png)

```
labelRangeMin.setBorder(BorderFactory.createEmptyBorder(0, 5, 0, 5));
labelRangeMin.setFont(new Font(Font.MONOSPACED, Font.PLAIN, 12));
labelRangeMax.setBorder(BorderFactory.createEmptyBorder(0, 5, 0, 5));
labelRangeMax.setFont(new Font(Font.MONOSPACED, Font.PLAIN, 12));
Dimension d = labelRangeMin.getPreferredSize();
d.width=60;
```

**Comment by i-make-robots** [comment]
Damnit, I thought I made enough room for 7 digits.

**Comment by i-make-robots** [comment]
@coliss86 added your fix in 75a47cc1.  looks good!
